backend/cellguide/pipeline/computational_marker_genes/utils.py

```diff
@@ -193,3 +91,69 @@ def query_gene_info_for_gene_description(gene_id: str) -> str:
         return data["name"]
     else:
         return gene_id
+
+
+@njit(parallel=True)
+def bootstrap_rows_percentiles(
```

Comment:
To confirm:
- We're bootstrapping _which cell types_ we're comparing to 
- We save the _final Nth percentile effect size_ for each gene, for each iteration
- We return the mean of these values

Is that right?

Reply:
Right!